### ISSUE TITLE
Resubmitting declaration returns - Disk size must be larger than current size

### ISSUE THREAD

**Issue by focrensh** [F5Networks/f5-declarative-onboarding#177]
### Environment
 * Declarative Onboarding Version: 1.15
 * BIG-IP Version: 14.1

### Summary
Resubmitting the declaration results in the error below. This error seems to happen the majority of the time, but not every time. Removing one of the provisioned modules  (asm) allows it to run. Adding back the module that was removed is also successful. The error seems to happen when resubmitting the declaration again after first success.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
        "schemaVersion": "1.0.0",
        "class": "Device",
        "async": true,
        "Common": {
            "class": "Tenant",
            "hostname": "bacon-bip.local",
            "dbvars": {
                "class": "DbVariables",
                "ui.advisory.enabled": true,
                "ui.advisory.color": "green",
                "ui.advisory.text": "Bacon EAST BIG-IP",
                "restjavad.useextramb": true,
                "provision.extramb": "500"
            },
            "myProvisioning": {
                "class": "Provision",
                "ltm": "nominal",
                "asm": "nominal",
                "avr": "nominal"
            },
            "analytics": {
                "class": "Analytics",
                "interval": 60,
                "offboxProtocol": "tcp",
                "offboxTcpAddresses": [
                    "127.0.0.1"
                ],
                "offboxTcpPort": 6514,
                "offboxEnabled": true,
                "debugEnabled": false
            }
        }
    }
```

2. Observe the following error response:
```json
"status": "ERROR",
 "message": "invalid config - rolled back",
 "errors": [
 "Disk size must be larger than current size.",
 "Disk size must be larger than current size."
 ],
 "result": {
 "class": "Result",
 "code": 500,
 "status": "ERROR",
 "message": "invalid config - rolled back",
 "errors": [
 "Disk size must be larger than current size.",
 "Disk size must be larger than current size."
 ]
 },
```

### Expected Behavior
Ok/Success on 2nd run.

### Actual Behavior
Error above

**Comment by dstokesf5** [comment]
Thank you for submitting this bug report. I have added it our internal product backlog as AUTOTOOL-1955.

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.17.0 release.

**Comment by mikeshimkus** [comment]
This issue seems to have resurfaces in DO v1.19.0.

2021-03-03T22:17:49.829Z [3481]: error: Task with error: {"code":500,"body":{"id":"090e4a44-347b-4cf6-ae1b-943e95347ed7","selfLink":"https://localhost/mgmt/shared/declarative-onboarding/task/090e4a44-347b-4cf6-ae1b-943e95347ed7","code":500,"status":"ERROR","message":"rollback failed","errors":["Disk size must be larger than current size.","Disk size must be larger than current size."],"result":{"class":"Result","code":500,"status":"ERROR","message":"rollback failed","errors":["Disk size must be larger than current size.","Disk size must be larger than current size."]},"declaration":{"schemaVersion":"1.0.0","class":"Device","async":true,"label":"my BIG-IP declaration for declarative onboarding","Common":{"class":"Tenant","hostname":"tf-func-test-vm-yrfjq8q4.test","myProvisioning":{"class":"Provision","ltm":"nominal","asm":"nominal"},"admin":{"class":"User","userType":"regular","shell":"bash","keys":[]},"dbvars":{"class":"DbVariables","provision.extramb":500,"restjavad.useextramb":true}}}}}

**Comment by dstokesf5** [comment]
The reappearance of this issue was tracked in #204 and was resolved as part of the 1.20.0 release.
